platform/linux/linux.cmake

```diff
@@ -8,6 +8,34 @@ find_package(X11 REQUIRED)
 
 pkg_search_module(LIBUV libuv REQUIRED)
 
+add_library(mbgl-platform-loop STATIC)
```

Comment:
You add this only to linux but other parts depend on it, so we should be a bit more careful I guess.

We should also make sure this does not interfere with the Qt build.

Reply:
FWIW it really shouldn't - it just introduces a new dep, `libuv`, but that's about it.